scripts/helm/install

```diff
@@ -16,7 +16,7 @@ kubernetesNamespace="ffc-demo"
     --atomic \
     --install \
     --namespace ${kubernetesNamespace} \
-    --values helm/artemis-values.yaml \
+    --values helm/ffc-demo-claim-service/artemis-values.yaml \
```

Comment:
Might be worth leaving `artemis-values.yaml` and `postgres-values.yaml` outside the `ffc-demo-claim-service` folder, since they're only used in development to deploy dependencies rather than being part of the service itself.

Reply:
We could keep them in there but add them to the .helmignore file so they won't be packaged?  Then we can still keep all the values files in one place.

Reply:
Yep, sounds good

Reply:
Done.  I've tested the package creation locally and confirmed those files are excluded.